### ISSUE TITLE
@MatrixParam in REST Dispatch

### ISSUE THREAD

**Issue by harry-peirse** [ArcBees/GWTP#619]
Hi there,
Any reason why Matrix Params aren't supported in REST Dispatch? I would be happy to create a PR for it, but I wanted to check if there was a special reason why it wasn't supported already?

**Comment by meriouma** [comment]
@Chris-V I believe you merged a PR for this?

**Comment by Chris-V** [comment]
Yes. It's only `@MatrixParam` though. Support for path segments will come later.
